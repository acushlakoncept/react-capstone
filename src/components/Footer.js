import React from 'react';

export default function Footer() {
  return (
    <footer className="footer">
      <div className="container">
        <span className="text-muted">
          &copy;
          { ` ${(new Date()).getFullYear()} All rights reserved | site by Acushla Koncepts.`}
        </span>
      </div>
    </footer>
  );
}
