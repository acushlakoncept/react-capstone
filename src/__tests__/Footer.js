import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('renders Footer correctly', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
