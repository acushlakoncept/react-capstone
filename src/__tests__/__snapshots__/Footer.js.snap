// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span
      className="text-muted"
    >
      Â©
       2020 All rights reserved | site by Acushla Koncepts.
    </span>
  </div>
</footer>
`;
